### PR TITLE
fix(web): editor URL setting

### PR DESCRIPTION
**Pull request by caichi-t**
# Overview
This PR makes it possible to config `editorUrl` in `reearth_config.json`.
If you set `http(s)://ANY_DOMAIN(/)` to the value, the `Go to Editor` button shows up.

### CODE REVIEW

File: web/src/components/molecules/Common/Header/index.tsx [hunk 1]
```diff
@@ -39,6 +39,10 @@ const HeaderMolecule: React.FC<Props> = ({
   const t = useT();
   const { logout } = useAuth();
   const navigate = useNavigate();
+  const url =
+    window.REEARTH_CONFIG?.editorUrl &&
+    currentWorkspace?.id &&
+    new URL(currentWorkspace.id, window.REEARTH_CONFIG?.editorUrl);
```
Comment (nourbalaha) [inline web/src/components/molecules/Common/Header/index.tsx:45]:
```suggestion
  const url = useMemo(() => {
    if (window.REEARTH_CONFIG?.editorUrl && currentWorkspace?.id) {
      return new URL(currentWorkspace.id, window.REEARTH_CONFIG?.editorUrl);
    }
    return undefined;
  }, [currentWorkspace?.id]);
```

File: web/src/config.ts [hunk 2]
```diff
@@ -15,21 +15,21 @@ export type Config = {
...
 const env = import.meta.env;
 
 export const defaultConfig: Config = {
   api: env.REEARTH_CMS_API || "/api",
-  auth0Audience: env.REEARTH_CMS_AUTH0_AUDIENCE || "https://api.test.reearth.dev",
-  auth0Domain: env.REEARTH_CMS_AUTH0_DOMAIN || "reearth-oss-test.eu.auth0.com",
-  auth0ClientId: env.REEARTH_CMS_AUTH0_CLIENT_ID || "k6F1sgFikzVkkcW9Cpz7Ztvwq5cBRXlv",
+  auth0Audience: env.REEARTH_CMS_AUTH0_AUDIENCE,
+  auth0Domain: env.REEARTH_CMS_AUTH0_DOMAIN,
+  auth0ClientId: env.REEARTH_CMS_AUTH0_CLIENT_ID,
```
Comment (nourbalaha) [inline web/src/config.ts:27]: Why did you remove the default values?

File: web/src/config.ts [hunk 3]
```diff
@@ -15,21 +15,21 @@ export type Config = {
...
   authProvider: env.REEARTH_CMS_AUTH_PROVIDER || "auth0",
   logoUrl: env.REEARTH_CMS_LOGO_URL,
   coverImageUrl: env.REEARTH_CMS_COVER_URL,
   cesiumIonAccessToken: env.REEARTH_CMS_CESIUM_ION_ACCESS_TOKEN || "",
-  webDomain: env.REEARTH_WEB_DOMAIN || "https://test.reearth.dev/",
+  editorUrl: env.REEARTH_CMS_EDITOR_DOMAIN,
```
Comment (nourbalaha) [inline web/src/config.ts:32]:
```suggestion
  editorUrl: env.REEARTH_CMS_EDITOR_URL,
```

**Review by nourbalaha — CHANGES_REQUESTED** [review]

File: web/src/config.ts (hunk 2 above)
Comment (caichi-t) [reply web/src/config.ts:27]: I discussed about this with Inoue-san, and he said the default values should not be made public since the test environment is not open for general users.

File: web/src/config.ts (hunk 3 above)
Comment (caichi-t) [reply web/src/config.ts:32]: I made a mistake... Thank you!!

File: web/src/config.ts (hunk 2 above)
Comment (nourbalaha) [reply web/src/config.ts:27]: Okay, then you can remove it.

File: web/src/components/molecules/Common/Header/index.tsx (hunk 1 above)
Comment (caichi-t) [reply web/src/components/molecules/Common/Header/index.tsx:45]: What do you think about the path (`dashboard/`), which could change depending on the environment?
Should I code it?

File: web/src/config.ts (hunk 3 above)
Comment (nourbalaha) [reply web/src/config.ts]: Thank you!

File: web/src/components/molecules/Common/Header/index.tsx (hunk 1 above)
Comment (caichi-t) [reply web/src/components/molecules/Common/Header/index.tsx:45]: Okay, I'll fix it. Thank you!

**Review by nourbalaha — APPROVED** [review]
